plugins/servicenow.rb

```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "ServiceNow"
+  authors [
+    "Ostorlab",
```

Comment:
Add a target that was detected with this script please.

Reply:
done